### PR TITLE
fix(components): Make countdown label smarter [JOB-39652]

### PR DESCRIPTION
**Pull request by robbyjobber**
## Motivations
- We want the countdown labels to be smarter by implementing smarter labeling of time units.

## Changes
- When there is a singular time left (i.e 1 hour, 1 day, 1 second), we will show a singular unit instead of a pluralized unit
- before:
image
- after:
image

## Testing
- Open up the Atlantis Playground > Countdown component > modify the time remaining 
- [ ] When the remaining time is not equal to `1` it should show a pluralized time unit (hours, days, etc)
- [ ] When the remaining time is equal to `1` > it should show a singular time unit (hour, day, etc)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by darryltec — APPROVED** [review]
It does change the pluralization

File: packages/components/src/Countdown/computeTimeUnit.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { computeTimeUnit } from "./computeTimeUnit";
+
+describe("computeTimeUnit", () => {
+  describe("when remainder time is not equal to one", () => {
+    it("should return a plural unit", () => {
+      expect(computeTimeUnit("0123", "day")).toEqual("days");
+      expect(computeTimeUnit("0", "minute")).toEqual("minutes");
```
Comment (adambobadam) [inline packages/components/src/Countdown/computeTimeUnit.test.tsx:7]: Nice.  Was curious about handling the 0 edge case
